examples/api/test/material/dropdown_menu/dropdown_menu.2_test.dart

```diff
@@ -10,7 +10,7 @@ import 'package:flutter_api_samples/material/dropdown_menu/dropdown_menu.2.dart'
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  testWidgets('DropdownMenu cursor behavoir', (WidgetTester tester) async {
+  testWidgets('DropdownMenu cursor behaviour', (WidgetTester tester) async {
```

Comment:
Can you change this to "behavior" (US english, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#spell-words-in-identifiers-and-comments-correctly)